tools/linter/adapters/pyfmt_linter.py

```diff
@@ -160,6 +160,7 @@ def run_ruff_format(content: str, path: Path) -> str:
                 "format",
                 "--config",
                 str(REPO_ROOT / "pyproject.toml"),
+                "--target-version=py39",
```

Comment:
NB: Setting `target-version` in `pyproject.toml` will also affect the `RUFF` linter and create huge diffs.

Reply:
Add a comment here to remove the target version pinning.

Reply:
I added a comment to `pyproject.toml`.